### ISSUE TITLE
[BUG-REPORT] typo in vaex-core/vaex/dataset.py

### ISSUE THREAD

**Issue by Jbollenbacher** [vaexio/vaex#1104]
In vaex=4.0.0a5 there is a typo in vaex/dataset.py at line 791. 

```
    def shape(self, column):
        return self.origina.shape(column)
```

should be 

```
    def shape(self, column):
        return self.original.shape(column)
```

**Comment by maartenbreddels** [comment]
Fixed in #1105, thanks.

**Comment by maartenbreddels** [comment]
Released, to install:
```
$ pip install "vaex-core>=4.0.0a11"
# or for the while kitchsink:
$ pip install "vaex>=4.0.0a6"
```
